Alignment/OfflineValidation/src/Trend.cc

```diff
@@ -22,10 +22,10 @@ namespace pt = boost::property_tree;
 
 Run2Lumi::Run2Lumi(fs::path file, int first, int last, float convertUnit)
     : firstRun(first), lastRun(last), convertUnit(convertUnit) {
-  cout << __func__ << endl;
+  //cout << __func__ << endl;
```

Comment:
leaving a lot of commented code is not a good option. Either remove or add a `verbose` flag in order to activate the `cout`s (by the way it's technically not thread-safe and in general `MessageLogger` is adivsed).